backend/infrahub/core/query/node.py

```diff
@@ -713,14 +714,24 @@ def __init__(self, ids: list[str], **kwargs: Any) -> None:
         super().__init__(**kwargs)
 
     async def query_init(self, db: InfrahubDatabase, **kwargs: Any) -> None:  # noqa: ARG002
+        self.params["ids"] = self.ids
         query = """
-        MATCH p = (root:Root)<-[r_root:IS_PART_OF]-(n:Node)
-        WHERE n.uuid IN $ids
+MATCH (n:Node)-[r:IS_PART_OF {status: "active"}]->(:Root)
+WHERE toString(n.uuid) IN $ids
```

Comment:
@ajtmccarty 
Sorry for digging this up but do you know why `toString()` was required? it's preventing the planner to use the index in my testing...

Reply:
Thanks, I will do the change in a PR I'm working on :+1: